### PR TITLE
Fix mpi tests

### PR DESCRIPTION
**Pull request by paulheinr**

### CODE REVIEW

File: src/mpi/controller.rs [hunk 1]
```diff
@@ -109,6 +110,17 @@ pub fn run(world: SystemCommunicator, config: Config) {
     let duration = end.sub(start).as_millis() / 1000;
     info!("#{rank} took: {duration}s");
 
+    info!("output dir: {:?}", config.output_dir);
+
+    if config.routing_mode == RoutingMode::AdHoc {
+        match remove_dir_all(config.output_dir + "routing") {
```
Comment (Janekdererste) [inline src/mpi/controller.rs:116]: I think this is a case for `expect`. Or do you really want to proceed if the tmp folder can't be removed?

File: src/mpi/messages.rs [hunk 2]
```diff
@@ -255,11 +255,11 @@ impl Plan {
                     result.acts.push(act);
                 }
                 IOPlanElement::Leg(io_leg) => match routing_mode {
-                    RoutingMode::AdHoc => {
+                    UsePlans => {
```
Comment (Janekdererste) [inline src/mpi/messages.rs:258]: This looks weird because one of the branches has the `RoutingMode` identifier and the other doesn't.

Comment (paulheinr) [reply src/mpi/messages.rs:258]: fixed it in #48

File: src/mpi/controller.rs (hunk 1 above)
Comment (paulheinr) [reply src/mpi/controller.rs:116]: fixed it in #48

**Pull request merged**